packages/klevu-ui/src/components/klevu-product/klevu-product.tsx

```diff
@@ -121,6 +121,18 @@ export class KlevuProduct {
    */
   @Prop() vatCaption?: string
 
+  /**
+   * To show the product code next to product name.
+   */
+  @Prop() showProductCode?: boolean
+  /**
+   * Caption to show if product is out of stock
+   */
+  @Prop() oosCaption?: string
```

Comment:
Let's not start using acronyms. `outOfStockCaption` is much better.

Reply:
renamed